examples/image_classification/mlflow_image_classification_peft_lora.py

```diff
@@ -33,9 +33,10 @@
                     help="Log metrics every X steps")
 parser.add_argument('--cache_dir', type=str, default=None, help='Directory to read/write data.')
 parser.add_argument("--amp", type=str, choices=["bf16", "fp16", "no"], default="no", help="Choose AMP mode")
-parser.add_argument('--checkpoint_dir', type=str, default="/nas/kimng/repo_clean/peft/examples/image_classification/swin-base-patch4-window7-224/10epoch.pt", help='Directory to save checkpoints.')
+parser.add_argument('--checkpoint_dir', type=str, default="/nas/peft/examples/image_classification/swin-base-patch4-window7-224/10epoch.pt", help='Directory to save checkpoints.')
```

Comment:
You don't need to put specific checkpoint_dir by default

Reply:
Agree. I have fixed it